example/android/build.gradle

```diff
@@ -1,14 +1,15 @@
 buildscript {
     ext {
-        buildToolsVersion = "34.0.0"
-
         // Android Installation Step 1a: Set compatible minSdkVersion and compileSdkVersion
         minSdkVersion = 23 // Minimum supported by Klaviyo Android SDK
         targetSdkVersion = 33 // Typically minSdkVersion <= targetSdkVersion <= compileSdkVersion
         compileSdkVersion = 34 // Klaviyo Android SDK is compiled against 34
-
-        ndkVersion = "25.1.8937393"
-        kotlinVersion = "1.8.0"
+        buildToolsVersion = "35.0.0"
+        minSdkVersion = 24
```

Comment:
Our SDK supports 23, but RN 0.77 is 24+

Reply:
so on native we support 23 but for RN it's only 24 onwards? NBD but checking.

Reply:
Well this is just the example app, since it was updated to 0.77, other RN packages probably won't work if I try to lower this to 23. I don't think it means the RN SDK will only support 24+, it'll just depend on the RN version of the host app.